_posts/2020-05-15-guard-duty-orgs.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: post
+title: Rolling Out Guard Duty Organisations
```

Comment:
generally when you use service names - try to use the full name and the Americanized spelling - e.g. AWS Organizations.

If you submit something to the AWS Blog one day they will make you do this.